packages/adapter-sveltekit/src/hooks/project-init.ts

```diff
@@ -200,6 +196,86 @@ const createSliceSimulatorPage = async ({
 	});
 };
 
+const createPreviewRouteMatcherFile = async ({
+	helpers,
+	options,
+}: SliceMachineContext<PluginOptions>) => {
+	const extension = await getJSFileExtension({ helpers, options });
+	const filename = path.join(`src/params/preview.${extension}`);
+
+	if (await checkHasProjectFile({ filename, helpers })) {
+		return;
+	}
+
+	const contents = source`
+		export function match(param) {
+			return param === 'preview';
+		}
+	`;
+
+	await writeProjectFile({
+		filename,
+		contents,
+		format: options.format,
+		helpers,
+	});
+};
+
+const createPreviewRouteDirectory = async ({
+	helpers,
+	options,
+}: SliceMachineContext<PluginOptions>) => {
+	const filename = path.join(
+		"src",
+		"routes",
+		"[[preview=preview]]",
+		"README.md",
+	);
+
+	if (await checkHasProjectFile({ filename, helpers })) {
+		return;
+	}
+
+	const contents = source`
+		This directory adds support for optional \`/preview\` routes. Do not remove this directory.
```

Comment:
@angeloashmore If I publish the preview doc tomorrow, can we link to it here?

https://prismic.io/docs/svelte-preview

Reply:
Yes, good idea. I added this line:

```md
See <https://prismic.io/docs/svelte-preview> for more information.
```